File: MaterialDesign/app/src/main/java/com/kmmx/materialdesign/MainActivity.java
package com.kmmx.materialdesign;

import android.os.Bundle;
import android.support.design.widget.CollapsingToolbarLayout;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.Toolbar;

public class MainActivity extends AppCompatActivity {

    Toolbar toolbar;
    CollapsingToolbarLayout collapsingToolbar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        toolbar = (Toolbar) findViewById(R.id.toolbar);
        collapsingToolbar = (CollapsingToolbarLayout) findViewById(R.id.collapsingToolbar);
        setSupportActionBar(toolbar);

        collapsingToolbar.setContentScrimColor(getResources().getColor(R.color.colorAccent));
    }
}

File: KotlinExample/app/src/main/java/com/kmmx/kotlinexample/MainActivity.kt
package com.kmmx.kotlinexample

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)


        toolbar.title = "titulo"

        setSupportActionBar(toolbar)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)

        val valor = 1.suma(2)
        val valorsuma = 1.resta(2)

    }

    infix fun Int.suma(add: Int): Int {
        return this + add
    }

    fun Int.resta(rest: Int) = this - rest

}
